ivy/container/base.py

```diff
@@ -5038,3 +5039,20 @@ def max_depth(self):
         if not kcs:
             return 0
         return max([len(kc.split("/")) for kc in kcs])
+    
+    @property
+    def is_variable(self):
```

Comment:
these docstrings don't make sense, a container will never be a variable, but it might contain variables at the leaves.